src/scss/lib/components/products-list.scss

```diff
@@ -0,0 +1,59 @@
+.products-list{
+  flex-direction: column;
+  .products-list-item{
+    position: relative;
+    overflow: hidden;
+    background-color: $product-background;
```

Comment:
How can I add an image as a background instead of plain color? I know that in the design it's without images, but I think all that space doesn't look good without an imager, isn't?

Reply:
You can insert pictures for the product using the tag img. Are you satisfied with this option?